--- components/opus/opus/silk/quant_LTP_gains.c	2021-11-04 18:11:35.479828356 +0100
+++ components/opus/opus/silk/quant_LTP_gains_old.c	2021-11-05 17:36:26.681279061 +0100
@@ -51,7 +51,7 @@
     const opus_int8      *cbk_ptr_Q7;
     const opus_uint8     *cbk_gain_ptr_Q7;
     const opus_int32     *XX_Q17_ptr, *xX_Q17_ptr;
-    opus_int32           res_nrg_Q15_subfr, res_nrg_Q15, rate_dist_Q7_subfr, rate_dist_Q7, min_rate_dist_Q7;
+    opus_int32           res_nrg_Q15_subfr, res_nrg_Q15=0, rate_dist_Q7_subfr, rate_dist_Q7, min_rate_dist_Q7;
     opus_int32           sum_log_gain_tmp_Q7, best_sum_log_gain_Q7, max_gain_Q7;
     opus_int             gain_Q7;
 
